source/edge-server/configure.txt

```diff
@@ -474,7 +493,7 @@ To cycle the Edge Server API key:
 
       .. code-block:: shell
 
-         edgectl --update-token=true
+         edgectl --update-registration-token
```

Comment:
should be `edgectl config --update-registration-token`

Reply:
more generally, `edgectl config` takes (almost) all the same args as edgectl init, and allows you to update an existing edge server deployment

after running edgectl config, you shuold run `edgectl stop` and `edgectl start`
(i'll probably get in an `edgectl restart` command in by monday)